test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -124,6 +137,22 @@ public LogoutRequestBuilder logoutUrl(String logoutUrl, Object... uriVars) {
 
 		private LogoutRequestBuilder() {
 		}
+
+		@Override public boolean isMergeEnabled() {
```

Comment:
Can you please put `@Override` on its own line to be consistent with Spring Security code conventions?

```suggestion
		@Override
		public boolean isMergeEnabled() {
```